### DISCUSSION TITLE
Notification sound for new mentions, replies etc

### DISCUSSION THREAD

**Discussion by seedy60** [NickColley/semaphore#56] [New features and ideas]
As an accessibility improvement for those who are blind/visually impaired, it'd be really cool if Semaphore included notification sounds for events such as new mentions, boosts, favourites, replies etc. These sounds could also be customized by the user so they aren't stuck with the default ones.

**Comment by glaroc2** [comment]
I know for me, it would be great because then I could know when new stuff came in on my hoem timeline or local or whatever, and also  know if someone  favorited my posts or replied or things like  that without having to go back every few minutes and check, that and it might come up visually, but since I'm blind, it doesn't actually do that for me so I have to press . sometimes in order for it to reload the timeline and these notification sounds would be really helpful in knowing when things were refreshed and especially when they were important to me, like for favorites, boosts, that sort of thing as mentioned in the OP.
